test/format_test.js

```diff
@@ -163,6 +163,50 @@ describe('format utility', function() {
     });
   });
 
+  describe('timeChangeInfo', function() {
+    it('should be a function', function() {
+      assert.isFunction(fmt.timeChangeInfo);
+    });
+
+    it('should error if 2 arguments are not passed', function() {
+      var err = 'You have not provided two datetime strings';
+      var x = '2014-01-01T01:00:00.000Z';
+      expect(fmt.timeChangeInfo.bind(fmt)).to.throw(err);
+      expect(fmt.timeChangeInfo.bind(fmt, x)).to.throw(err);
+    });
+
+    it('should return an object containing strings of times when both are on same day', function() {
+      var x = '2014-01-01T01:00:00.000Z';
+      var y = '2014-01-01T04:00:00.000Z';
```

Comment:
Your `x` and `y` are not the format that `from` and `to` time changes come in. The `from` and `to` are ISO-8601-formatted but *without any offset information* (because they are purely relative device time) - that is, `2014-01-01T01:00:00`. There is some weirdness with how different browsers and libraries (JS Date vs. moment) deal with dates that do and don't actually appear as UTC with the Zulu-time `Z` suffix or a timezone offset, so I'd vastly prefer if the tests used data that is identical to the source. Can you adjust? I *think* these are all going to pass even in the naive format (especially in Chrome), but it would be good to feel more certain :)